internal/characters/nilou/attack.go

```diff
@@ -69,3 +69,15 @@ func (c *char) Attack(p map[string]int) action.ActionInfo {
 		State:           action.NormalAttackState,
 	}
 }
+
+func (c *char) attackPattern(attackIndex int) combat.AttackPattern {
```

Comment:
instead of a function, pls use a slice for this. everything except for the radius is the same.